source/WorkingPractices/approvals.rst

```diff
@@ -40,6 +38,9 @@ A list of current code owners for each project can be found in
     able to guide your change to fit in with the bigger picture of what is
     happening.
 
+Code Owners are good candidates for choosing a SciTech reviewer, and you should
```

Comment:
Do you mean "... for choosing **as** a SciTech reviewer"
or perhaps "... for helping select a suitable SciTech Reviewer"